### PR TITLE
Update ember and friends

### PR DESCRIPTION
**Pull request by jherdman**

### CODE REVIEW

**Comment by jherdman** [comment]
This is awaiting cibernox/ember-power-calendar#249

**Comment by jherdman** [comment]
@cibernox ready!

**Comment by jherdman** [comment]
Hmm... Travis isn't running here either. This _should_ go green. I say "should" because of the ongoing `is-promise` fiasco.

**Comment by jherdman** [comment]
@cibernox have you had a chance to look at this?

**Comment by jherdman** [comment]
@cibernox sorry to nag, but do you have time to take a look at this?

File: .template-lintrc.js [hunk 1]
```diff
@@ -1,5 +1,9 @@
 'use strict';
 
+const pending = [
+];
+
 module.exports = {
-  extends: 'recommended'
+  extends: 'octane',
+  pending
```
Comment (cibernox) [inline .template-lintrc.js:8]: what's this pending thing?

**Comment by cibernox** [comment]
Merged! Thank you for working on this update. I let it sit for a day or two and release to NPM a new version then.

File: .template-lintrc.js (hunk 1 above)
Comment (jherdman) [reply .template-lintrc.js:8]: `pending` is a list of violations that haven't been resolved. I probably should have removed this given that there are none. Historically I've kept this in my apps/addons to make future diffs easier to read when new violations pop up.

**Comment by jherdman** [comment]
Many thanks, good sir!

**Comment by cibernox** [comment]
Released 0.8! :tada:

**Comment by jherdman** [comment]
Many thanks again! ❤️

**Pull request merged**